elasticsearch/run.sh

```diff
@@ -100,8 +104,30 @@ rm -f $LOG_FILE
 exit 1
 }
 
+seed_searchguard(){
+    /usr/share/elasticsearch/plugins/search-guard-2/tools/sgadmin.sh \
+        -cd ${HOME}/sgconfig \
+        -i .searchguard.${HOSTNAME} \
```

Comment:
I do not believe so.  I think it simply updates the index.  The log produces for each file in the directory:

```
Will update 'internalusers' with /opt/app-root/src/sgconfig/sg_internal_users.yml
SUCC: Configuration for 'internalusers' created or updated
```